backend/accounts.sql

```diff
@@ -0,0 +1,13 @@
+-- @block 
+CREATE TABLE IF NOT EXISTS account (
+    accountID INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    email VARCHAR(255) NOT NULL,
+    encryptedPass VARCHAR(255) NOT NULL,
```

Comment:
If we're using `SHA256` encryption, we would only need `numCharacters = 256 (bits) / numBitsInRepresentation`. For example, if we're going to represent the encrypted-password in hexadecimal (4 bits per digit), we'd only need `256 / 4 = 64` characters.

Reply:
I'm not really familiar with how encryptions work but what if we decide to us e different encryption? Or is that extra space not necessary for any kind of encryption.

Reply:
Different encryption algorithms will produce varying output lengths. We can leave this as-is for now and revisit once we know what's required. If the encryption algorithm we use produces an output longer than 255 characters, things will break here.